plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduIntegrationDateColumns.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
No need to add a new class. Please update the existing test class. e.g. BaseKuduConnectorSmokeTest

Reply:
The existing class are handling specific type such as Integer or Decimal. To keep code clear I build dedicated class on Date only.

Reply:
This is different from how we manage test code. Please use the existing test class.